src/App.tsx

```diff
@@ -0,0 +1,53 @@
+import React, {useEffect, useState} from 'react';
+import './App.css';
+import RecipeCard from "./components/RecipeCard";
+import { Recipe } from "./Interface";
+
+const App = () => {
+  const APP_ID = "43c84aae";
+  const APP_KEY = "ccf75b2d7f25b9a49fcc176c4cc78068";
+
+  const [recipes, setRecipes] = useState([]);
+  const [search, setSearch] = useState("");
+  const [query, setQuery] = useState("chicken");
+
+  useEffect( () => {
+    getRecipes();
+  }, [query]);
+
+  const getRecipes = async (): Promise<void> => {
+    const response = await fetch(`https://api.edamam.com/search?q=${query}&app_id=${APP_ID}&app_key=${APP_KEY}`);
+    const data = await response.json();
+    setRecipes(data.hits);
+    console.log(data.hits);
```

Comment:
Usually your PL will mention to remove `console.log` statements in your commits or your linter will catch your console.log statements before you commit using "pre-commit" hooks, which you can think of as scripts that run and check your code before it's committed.  This is totally fine for our onboarding activity 👍